### PR TITLE
feat: add Coder template to simplify contributing environment

### PR DESCRIPTION
**Pull request by bpmct**
To try it out and see what is left to do, check out the [README](https://github.com/cdr/docs/tree/bpmct/add-coder/.coder#readme)

### CODE REVIEW

**Comment by bpmct** [comment]
haven't forgotten about this :( planning on getting to it this week

**Comment by bpmct** [comment]
Yikes this PR has been open wayyyy too long. I figured out how to get rewrap and other extensions working by installing from open-vsx. @khorne3 whenever you get the chance - a re-test would be awesome

File: .coder/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+With sandbox.coder.com:
+
+[![Open in
+Coder](https://cdn.coder.com/embed-button.svg)](https://sandbox.coder.com/wac/build?project_oauth_service=github&template_oauth_service=github&template_ref=bpmct%2Fadd-coder&template_url=https://github.com/cdr/docs)
+
+Your own deployment:
+
+```text
+Add image: bencdr/coder:latest (DockerHub)
```
Comment (khorne3) [inline .coder/README.md:13]:
```suggestion
Import image: bencdr/coder:latest (DockerHub)
```

Also, did you make this image private?

File: .coder/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+New workspace > Create from template
```
Comment (khorne3) [inline .coder/README.md:15]:
```suggestion
New workspace > New workspace from template
```

File: .coder/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Repo url: https://github.com/cdr/docs
```
Comment (khorne3) [inline .coder/README.md:17]:
```suggestion
Git repository URL: https://github.com/cdr/docs
```

**Comment by bpmct** [comment]
Was referencing the wrong issue, fixed & committed changes.

**Review by khorne3 — APPROVED** [review]

**Comment by bpmct** [comment]
all good to merge, pending adding access tokens

**Pull request merged**